python/ray/train/v2/api/train_fn_utils.py

```diff
@@ -186,7 +186,9 @@ def train_func(config):
 
 @PublicAPI(stability="alpha")
 @requires_train_worker()
-def get_all_reported_checkpoints() -> List["ReportedCheckpoint"]:
+def get_all_reported_checkpoints(
+    view: CheckpointView = CheckpointView.VALIDATED,
```

Comment:
I thought the arg name view was a bit confusing, since it doesn't really indicate the waiting aspect.

What about `wait_for_checkpoint_state`, or `consistency_mode`?